pkg/render/render.go

```diff
@@ -169,17 +169,22 @@ func translate(l *gotext.Locale, key string, vars ...interface{}) (string, error
 
 func templateFuncs() htmltemplate.FuncMap {
 	return map[string]interface{}{
-		"joinStrings":    strings.Join,
-		"trimSpace":      project.TrimSpace,
-		"stringContains": strings.Contains,
-		"toLower":        strings.ToLower,
-		"toUpper":        strings.ToUpper,
-		"safeHTML":       safeHTML,
-		"selectedIf":     selectedIf,
-		"t":              translate,
+		"joinStrings":      strings.Join,
+		"trimSpace":        project.TrimSpace,
+		"stringContains":   strings.Contains,
+		"toLower":          strings.ToLower,
+		"toUpper":          strings.ToUpper,
+		"safeHTML":         safeHTML,
+		"selectedIf":       selectedIf,
+		"t":                translate,
+		"passwordSentinel": pwdSentinel,
```

Comment:
I think this will bring it in as a func, not a variable (e.g. `passwordSentinel` vs `.passwordSentinel`) - all templates need to be updated.

Reply:
... I did that

Reply:
Weird - I think GitHub was only showing me a partial diff.